common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2089,6 +2093,16 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(0)
 
+  val NETWORK_IO_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.<module>.io.threads")
+      .categories("network")
+      .doc(s"Default number of threads used in the server and client thread pool. " +
+        s"This specifies thread configuration based on JVM's allocation of cores. " +
+        s"If setting <module> to `${TransportModuleConstants.DATA_MODULE}`, " +
+        s"it works for shuffle client push and fetch data.")
+      .intConf
+      .createWithDefault(8)
```

Comment:
seems behavior change

Reply:
@turboFei, there is no behavior change. Just add the config option in `CelebornConf` with the default value `8`.